docs: use helm charts instead of manifests + document KeptnConfig CRD (#747)
keptn/lifecycle-toolkit: docs/content/en/docs/snippets/tasks/install.md

## Install version 0.6.0 and above

> In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts:

Reviewer: ```suggestion
In version 0.6.0 and later, you can install the Lifecycle Toolkit using Helm charts:
```

Reviewer 2: I would start the section by telling the user that we have two options
```suggestion
In version 0.6.0 and later, you can install the Lifecycle Toolkit using Helm charts or manifests.

For installing the Lifecycle Toolkit via Helm chart:
```

```
helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
helm repo update
helm upgrade --install keptn-lifecycle-toolkit keptn/lifecycle-toolkit -n keptn-lifecycle-toolkit-system --create-namespace --wait
```

> To install a specific version use `--version <version>` as part of `helm upgrade --install` command.

Reviewer: ```suggestion
To install a specific version, use the `--version <version>` flag as part of `helm upgrade --install` command.
```

To list available versions:

```
helm repo update
helm search repo keptn-lifecycle-toolkit
```

The `helm upgrade --install` command offers a flag called `--set`, which can be used to specify configuration options using the format key1=value1,key2=value2,....
The full list of available flags can be found in the [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/README.md).

> For installing the Lifecycle Toolkit via manifests use:

Reviewer: ```suggestion
All versions of the Lifecycle Toolkit can be installed using manifests, with a command like the following:
```

Reviewer: What would you think about moving the info about installing from manifests under the 0.5.0 then adding a note that later versions can also use this method?

## Install version 0.6.0 and above

In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts,
or manifests.

For installing the Lifecycle Toolkit via Helm chart:

```shell
helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
helm repo update
helm upgrade --install keptn-lifecycle-toolkit keptn-lifecycle-toolkit/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
```

To install a specific version, use the `--version <version>` falg as part of
`helm upgrade --install` command.

To list available versions:

```shell
helm repo update
helm search repo keptn-lifecycle-toolkit
```

The `helm upgrade --install` command offers a flag called `--set`, which can be used to specify
configuration options using the format key1=value1,key2=value2,....
The full list of available flags can be found in the [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/helm/chart/README.md).

> 

Reviewer 3: Or you could download the chart value file and modify it using
`helm get values RELEASE_NAME [flags] > values.yaml`
https://helm.sh/docs/helm/helm_get_values/

and install adding --values=values.yaml

## Install version 0.6.0 and above

In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts,
or manifests.

For installing the Lifecycle Toolkit via Helm chart:

```shell
helm repo add klt https://charts.lifecycle.keptn.sh
helm repo update
helm upgrade --install klt keptn-lifecycle-toolkit/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
```

> To install a specific version, use the `--version <version>` falg as part of

Reviewer 4: ```suggestion
To install a specific version, use the `--version <version>` flag as part of
```

The Lifecycle Toolkit and its dependencies are now installed and ready to use.

> **Note:** Installation of the Lifecycle Toolkit version 0.5.0 and lower is not supported via helm charts.

Reviewer: ```suggestion
**Note:** Versions 0.5.0 and lower of the Lifecycle Toolkit can only be installed using the manifest.  They cannot be installed from a Helm charts.
```

Reviewer 2: For the `**note**` part, please look at the suggestion of this PR: https://github.com/keptn/lifecycle-toolkit/pull/962#pullrequestreview-1328233789
